### DISCUSSION TITLE
filterValue() doesn't work

### DISCUSSION THREAD

**Discussion by bai1hao** [orchidsoftware/platform#2740] [Q&A]
` TD::make('id')->filterValue(function ($value) {
                    $user = Good::find($value);
                    return $user->name;
}),
`

![image](https://github.com/orchidsoftware/platform/assets/93936033/dd982680-e8f8-4e51-b4ec-cf62d4dface8)

I've already change the return like return '1',but the value is still id.

**Comment by macit-emre** [comment]
you may use filter method. filterValue is designed for dropdowns (as i guest) that when you select a name and for example its value is 3, then, filtered th shows us as we search for "3". you may use filterValue here to make it show the meaning of "3"

you can see filter usage in documentation of orchid website
